doc/intro.rst

```diff
@@ -345,16 +345,17 @@ Tango device
 
 .. code-block:: bash
 
-    $ tango-yaml tango_device_name -h
+    $ tango-yaml tango_device -h
 
-    usage: tango_yaml tango_device [-h] tango_device_name
+      usage: tango_yaml tango_device [-h] tango_device_name
 
-    positional arguments:
-    tango_device_name  Tango device name in the format domain/family/member.
-                        TANGO_HOST env variable has to be set
+      positional arguments:
+        tango_device_name  Tango device name in the domain/family/member format or
+                          the FQDN
```

Comment:
I don't know if it matters, but I see that these sentences are not inline.

Reply:
Yeah, argparse does that formatting.